utils/src/server/rpc_module.rs

```diff
@@ -349,96 +350,58 @@ impl Methods {
 	/// Helper to call a method on the `RPC module` without having to spin up a server.
 	///
 	/// The params must be serializable as JSON array, see [`ToRpcParams`] for further documentation.
-	pub async fn call_with<Params: ToRpcParams>(&self, method: &str, params: Params) -> Option<String> {
-		let params = params.to_rpc_params().ok();
-		self.call(method, params).await
-	}
-
-	/// Helper alternative to `execute`, useful for writing unit tests without having to spin
-	/// a server up.
-	pub async fn call(&self, method: &str, params: Option<Box<RawValue>>) -> Option<String> {
-		let req = Request::new(method.into(), params.as_deref(), Id::Number(0));
-
-		let (tx, mut rx) = mpsc::unbounded();
-		let sink = MethodSink::new(tx);
-
-		if let MethodResult::Async(fut) = self.execute(&sink, req, 0) {
-			fut.await;
-		}
-
-		rx.next().await
-	}
-
-	/// Perform a "in memory JSON-RPC method call" and receive further subscriptions.
-	/// This is useful if you want to support both `method calls` and `subscriptions`
-	/// in the same API.
-	///
-	/// There are better variants than this method if you only want
-	/// method calls or only subscriptions.
-	///
-	/// See [`Methods::test_subscription`] and [`Methods::call_with`] for
-	/// for further documentation.
-	///
-	/// Returns a response to the actual method call and a stream to process
-	/// for further notifications if a subscription was registered by the call.
 	///
-	/// ```
-	/// #[tokio::main]
-	/// async fn main() {
-	///     use jsonrpsee::RpcModule;
-	///     use jsonrpsee::types::{
-	///         EmptyParams,
-	///         v2::{Response, SubscriptionResponse}
-	///     };
-	///     use futures_util::StreamExt;
-	///
-	///     let mut module = RpcModule::new(());
-	///     module.register_subscription("hi", "hi", "goodbye", |_, mut sink, _| {
-	///         sink.send(&"one answer").unwrap();
-	///         Ok(())
-	///     }).unwrap();
-	///     let (resp, mut stream) = module.call_and_subscribe("hi", EmptyParams::new()).await.unwrap();
-	///     assert!(serde_json::from_str::<Response<u64>>(&resp).is_ok());
-	///     let raw_sub_resp = stream.next().await.unwrap();
-	///     let sub_resp: SubscriptionResponse<String> = serde_json::from_str(&raw_sub_resp).unwrap();
-	///     assert_eq!(&sub_resp.params.result, "one answer");
-	/// }
-	/// ```
```

Comment:
This example was good to have there I think. Can we port it over to the new API.

Reply:
sure